frontend/leaderboard/src/Metrics.re

```diff
@@ -109,9 +109,12 @@ let calculateSnarkFeeCount = (map, block: Types.NewBlock.data) => {
              switch (feeCount) {
              | Some(feeCount) =>
                let result =
-                 Int64.add(
-                   Int64.of_string(snarkJob.fee),
-                   Int64.of_string(feeCount),
+                 Int64.div(
```

Comment:
should we round fairly instead of always round down? I think you can do this by adding a constant `500M` before dividing.

Reply:
Good catch! Didn't realize `Int64.div` worked as such. It should be fixed now. :)

Reply:
@bkase what? I don't see how rounding fairly applies in this scenario
That said, I don't think we should be doing the conversion at this stage. Makes more sense to add everything up and then set the thresholds to be in nanocodas. Otherwise if the fee is < 1 coda, it will never add up? Unless I'm misreading this.

While we're at it, can we rename calculateSnarkFeeCount->calculateSnarkFeeSum and feeCount->feeSum? since it's not the count of how many fees there are.